web/src/components/FieldEntryAlert.tsx

```diff
@@ -25,6 +26,9 @@ export const FieldEntryAlert = (props: Props): ReactElement => {
             {field.children}
           </li>
         ))}
+        {props.additionalErrors?.map((error) => {
+          return <li key={error}>{error}</li>;
+        })}
```

Comment:
I'm looking at this again, and I think I'm a little confused. Right now this error message seems to be breaking the current implementation. We already have a "signers" error. Is this new message supposed to be a detail associated with the current bullet? If so should we be implementing this change as a sort of a nested list structure?

If this is going to be a standalone error, should it drive to the location on the page where we're missing information?
<img width="606" alt="Screenshot 2023-09-11 at 2 30 07 PM" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22647707/4b1a470a-fa56-4347-97f9-8ff4c746bb69">

Reply:
I'm looking at the design and it does look like this is the intended structure. I feel like this should probably drive to the field where the error is present.